configs/efficientdet_d0.yml

```diff
@@ -39,7 +37,7 @@ EfficientHead:
   alpha: 0.25
   delta: 0.1
   output_decoder:
-    score_thresh: 0.05   # originally 0.
+    score_thresh: 0.0
```

Comment:
would like some ablation experiments, what is the mAP like when `sync_bn` and `score_thresh` is unchanged?

Reply:
Ablation experiments have been done. Without the lower score_thresh and the sync_bn, the mAP could only achieve 0.335 on coco minival.
![image](https://user-images.githubusercontent.com/22272184/93572172-091ea480-f9c8-11ea-9f21-424f87d16a9e.png)

Reply:
so basically all the gain in mAP is from these 2 configuration change?

Reply:
based on the modified bifpn

Reply:
I have to point out **again**, ur bifpn is not aligned with the official tensorflow implementation, as well as the drop connected in efficientnet backbone. I think it will be helpful for EfficientDet-D1 to D7x. If ur team confirm it is useless, I think you just hit that I could withdraw the submission.

Reply:
my point is, neither of these settings change are free,  will increase either training time or inference latency, and with the bifpn change alone (which also adds additional cost, albeit small) , mAP actually dropped 0.3, am I understanding it correctly?